.github/workflows/publish_images.yaml

```
@@ -0,0 +1,29 @@
+name: Build and publish pulp-fixture OCI Image
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install requirements
+        run: |
+          pip install pylint
+          sudo apt-get install -y shellcheck
+      - name: Build & test the image
+        run: |
+          .ci/build_image.sh
+      - name: Display log on error
+        if: failure()
+        run: docker logs pulp
+      - name: Push image to registries
+        env:
+          DOCKER_BOT_PASSWORD: ${{ secrets.DOCKER_BOT_PASSWORD }}
+          DOCKER_BOT_USERNAME: ${{ secrets.DOCKER_BOT_USERNAME }}
```

We don't have a secret for this. Looks like we just [hardcode it](https://git.io/Jkcs0).

---

This is taken from OCI-images.

---

https://github.com/pulp/pulp-oci-images/blob/master/.github/workflows/publish_images.yaml#L23

---

Looks like they set it for the repo. I'm leaning towards hardcoding it though since it seems pointless to have it as a secret. wdyt?

---

I don't have a strong opinion.
```suggestion
          DOCKER_BOT_USERNAME: pulpbot
```

---

LGTM, I switched pulp-oci-images to use our organization's DOCKER_BOT_PASSWORD secret.